pkgs/top-level/all-packages.nix

```diff
@@ -17191,6 +17193,12 @@ in
 
   cligh = python3Packages.callPackage ../development/tools/github/cligh {};
 
+  clight = callPackage ../applications/misc/clight { };
+
+  clightd = callPackage ../applications/misc/clight/clightd.nix {
+    inherit (xorg) libXext libXdmcp libXrandr libpthreadstubs;
```

Comment:
Xorg packages are available in the argument list by default, so you can remove this line completely and it will still work.